### ISSUE TITLE
Add `--force-FRI` command line flag to main `dvmdostem` executable.

### ISSUE THREAD

**Issue by tobeycarman** [uaf-arctic-eco-modeling/dvm-dos-tem#673]
It would be nice to implement this similar to the `--force-cmt` command line flag.

**Comment by hgenet** [comment]
That would be great to facilitate site-specific investigations of changes in fire frequency!!!!

**Comment by tobeycarman** [comment]
Any ideas what a clean interface would look like for control over the FRI in different stages? How necessary is this?

**Comment by hgenet** [comment]
Yeah - although changes in fire frequency is mostly focused on historical and/or scenario, it could also be tested at eq and/or sp stages.
